client/client-multi/client-cpp/impl/KaaClient.cpp

```diff
@@ -262,14 +259,28 @@ void KaaClient::initKaaTransport()
     notificationManager_->setTransport(std::dynamic_pointer_cast<NotificationTransport, INotificationTransport>(notificationTransport));
 #endif
 #ifdef KAA_DEFAULT_BOOTSTRAP_HTTP_CHANNEL
-    bootstrapChannel_.reset(new DefaultBootstrapChannel(channelManager_.get(), *clientKeys_, status_));
+    bootstrapChannel_.reset(new DefaultBootstrapChannel(channelManager_.get(), *clientKeys_, context_));
     bootstrapChannel_->setDemultiplexer(syncProcessor_.get());
     bootstrapChannel_->setMultiplexer(syncProcessor_.get());
     KAA_LOG_INFO(boost::format("Going to set default bootstrap channel: %1%") % bootstrapChannel_.get());
     channelManager_->addChannel(bootstrapChannel_.get());
 #endif
+#ifdef KAA_DEFAULT_OPERATION_HTTP_CHANNEL
```

Comment:
Please, remove both HTTP and HTTP LP channels from a startup configuration.
A user should add them manually.